tendrl-api.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
 Description=Tendrl Api Daemon
+Documentation=https://github.com/Tendrl/api
```

Comment:
Is this the correct path or we should refer `https://github.com/Tendrl/api/docs`